tfjs-backend-wasm/src/kernels/_FusedMatMul.ts

```diff
@@ -103,7 +103,7 @@ function fusedBatchMatMul(args: {
   return out;
 }
 
-export const fusedMatMulConfig: KernelConfig = {
+export const _fusedMatMulConfig: KernelConfig = {
   kernelName: _FusedMatMul,
```

Comment:
Not related with your change. But I am curious why `_FusedMatMul` starts with `_`. @pyu10055 Do you know the reason?